doc/building.md

```diff
@@ -56,7 +56,7 @@ Dependency list:
 ### Prerequisite steps for Ubuntu users (Ubuntu 15.04)
 
  - `sudo apt-get update`
- - `sudo apt-get install cmake libfreetype6-dev python3-dev libepoxy-dev libsdl2-dev libsdl2-image-dev libopusfile-dev libfontconfig1-dev libharfbuzz-dev opus-tools python3-pil python3-numpy python3-pygments python3-pip qtdeclarative5-dev qml-module-qtquick-controls`
```

Comment:
If we make use if libopus directly, I'd add it, who knows how this may be packaged in the future. We can't avoid the explicit avoiding, developers will need to install those packages, unfortunately.